src/meta/src/hummock/compaction/mod.rs

```diff
@@ -182,7 +182,10 @@ impl CompactStatus {
         if task.input_ssts[1].level_idx == task.target_level
             && task.input_ssts[1].table_infos.is_empty()
         {
-            return true;
+            return task.input_ssts[0]
+                .get_table_infos()
+                .iter()
+                .all(|info| info.get_divide_version() == 0);
```

Comment:
If `divide_version` of a table is not zero, it means that it is splitted from the other group, isn't it?
So why we can not move a shared sst ? I think it does not affect

Reply:
> If `divide_version` of a table is not zero, it means that it is splitted from the other group, isn't it?
> So why we can not move a shared sst ? I think it does not affect

yes. it does not affect. Since you think that keeping it is ok, I will revert here.